merlion/models/forecast/trees.py

```diff
@@ -110,70 +94,62 @@ def _default_train_config(self):
         return dict()
 
     def _train(self, train_data: pd.DataFrame, train_config=None):
+        # if univariate, or prediction_stride is 1 for multivariate, call autoregressive model
+        if self.dim == 1 or self.prediction_stride == 1:
+            return super()._train(train_data, train_config)
+
+        logger.info(
+            f"Model is working on a multivariate dataset with prediction_stride > 1, "
+            f"default multi-output regressor training strategy will be adopted "
+            f"with prediction_stride = {self.prediction_stride} "
+        )
+
         train_data = TimeSeries.from_pd(train_data)
```

Comment:
Maybe you can update the `RollingWindowDataset` so it accepts either `TimeSeries` or `pd.DataFrame` as input? You could just do a type check.

Reply:
good idea, will do to improve the efficiency to avoid unnecessary data transformation in the middle

Reply:
Done